### PR TITLE
Sofia

### PR DESCRIPTION
**Pull request by sofiahaglund**
Removed the depreciated pages, changed the dropdown menus of "production" and "projects" to that of the screenshots in Trello, changed the default date of the slider in the "applications" page to one year ago, deleted the "workset setup" page and the link to the UPPMAX quotas

### CODE REVIEW

**Comment by ewels** [comment]
> **Title:** _Sofia_
> **Description:** _No description provided._

Please let us know what you are changing in pull requests 😉

**Comment by sofiahaglund** [comment]
oh sorry, thought that the commit messages would be enough :-) I'll add a description

**Comment by ewels** [comment]
The commit messages are good :) But it's just good practice to always write a description I think. Maybe that's just my overly verbose English nature though (I struggle to write any email that's shorter than half a page of A4).

File: run_dir/design/base.html [hunk 1]
```diff
@@ -51,30 +51,34 @@
...
+                    <li><a href="/production/cronjobs">Cronjobs List</a></li><li role="separator" class="divider"></li>
+                    <li class="dropdown-header">Flowcells</li>
+                    <li><a href="/flowcells_plot">Flowcells Trends</a></li>
+                    <li><a href="/flowcell_count_plot">Flowcells Numbers</a></li><li role="separator" class="divider"></li>
+                    <li class="dropdown-header">Tools</li>
+                    <li><a href="/nas_quotas">NAS Quotas</a></li>
+                    <li><a href="/reads_total/">Read Count Totals</a></li>
+                    <li><a href="/proj_meta">Compare Project Meta</a></li>
+                    <li><a href="/instrument_logs">View Instrument Logs</a></li>
+                    </li><li role="separator" class="divider"></li>
```
Comment (ewels) [inline run_dir/design/base.html]: Looks like an extra `</li>` crept in here..

I try to keep HTML lists `<li>`s indented from the `<ul>`, and with each `<li>` element on its own line (eg. including the dividers). Makes it easier to read..

File: run_dir/design/applications.html [hunk 2]
```diff
@@ -65,8 +65,10 @@ <h2>Samples per Application</h2>
 <script src="/static/nvd3/lib/d3.v2.js"></script>
 <script src="/static/nvd3/nv.d3.js"></script>
 <script type="text/javascript">
-var start = new Date(2012,0,1);
+
 var stop = new Date();
+var start = new Date(stop.getFullYear() - 1, stop.getMonth(), stop.getDate());
```
Comment (ewels) [inline run_dir/design/applications.html]: Great! 👍

File: run_dir/design/base.html [hunk 3]
```diff
@@ -51,30 +51,34 @@
...
+                    <li><a href="http://lims-dashboard.scilifelab.se">Script Dashboard</a></li>
                     </ul>
-                </li>
-                <li class="dropdown" id="projects">
+                    </li>
```
Comment (ewels) [inline run_dir/design/base.html]: Indentation again here would make this easier to understand..

File: run_dir/design/base.html [hunk 4]
```diff
@@ -51,30 +51,34 @@
...
                     <ul class="dropdown-menu" role="menu">
-                        <li><a href="/projects/all">All</a></li>
-                        <li><a href="/projects/pending">Pending</a></li>
-                        <li><a href="/projects/reception_control">Reception Control</a></li>
-                        <li><a href="/projects/ongoing">Ongoing</a></li>
-                        <li><a href="/projects/pending_review">Needs Review</a></li>
-                        <li><a href="/projects/closed">Closed</a></li>
-                        <li><a href="/projects/aborted">Aborted</a></li>
-                    </ul>
+                  <li><a href="/projects/all">All</a>
```
Comment (ewels) [inline run_dir/design/base.html]: Indentation again 😄 Sorry, I keep thinking that the HTML is messed up then realise that it's fine, which is why I've mentioned it each time.

File: status/samples.py [hunk 5]
```diff
@@ -0,0 +1,231 @@
+"""Handlers related with samples
+"""
+
+import tornado.web
+import json
+import cStringIO
+
+import matplotlib.gridspec as gridspec
```
Comment (ewels) [inline status/samples.py]: What are these files? Are they meant to be here?

File: status/sequencing.py [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+
+    return InstrumentSeriesDataHandler
+
+
+InstrumentClusterDensityDataHandler = make_instrument_series_handler("instrument/clusters_raw")
+
+
+class InstrumentClusterDensityPlotHandler(InstrumentClusterDensityDataHandler):
+    """ Gives a plot for series of cluster densities for instruments, over time.
+    Loaded through /api/v1/instrument_cluster_density.png url
```
Comment (ewels) [inline status/sequencing.py]: This one also looks like an old script that has been previously deleted I think..?

File: run_dir/design/base.html (hunk 4 above)
Comment (sofiahaglund) [reply run_dir/design/base.html]: sorry, once again I'm just being sloppy!

File: status/samples.py (hunk 5 above)
Comment (sofiahaglund) [reply status/samples.py]: I accidentally removed the samples.py file, and then just put it back (the whole file without having edited it) so I have not even looked at those lines, but I will check..

**Comment by sofiahaglund** [comment]
Senthil and I have now deployed it on stage, and it works! :-)

**Comment by sofiahaglund** [comment]
Deployed!

**Pull request merged**